packages/lib/getStampFyiURL.ts

```diff
@@ -7,6 +7,10 @@ import urlcat from 'urlcat';
  * @returns The cdn.stamp.fyi URL.
  */
 const getStampFyiURL = (address: string): string => {
+  // race condition fix when loging out
+  if (!address) {
+    return '';
```

Comment:
you can return stamp url with null address aka `export const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';`